pipelines/pipelines/cli/build-branch.yaml

```diff
@@ -478,18 +485,64 @@ spec:
       value: 
       - pipeline
       - start
-      - cli-snapshot-main-to-prod
+      - branch-isolated
       - -n
       - galasa-build
       - --prefix-name 
-      - cli-snapshot
+      - trigger-cli-main
       - --use-param-defaults
       - --workspace
       - name=git-workspace,volumeClaimTemplateFile=./pipelines/templates/git-workspace-template.yaml
       - --pod-template
       - ./pipelines/templates/pod-template.yaml
       - --serviceaccount
       - galasa-build-bot
+    workspaces:
+     - name: git-workspace
+       workspace: git-workspace  
+
+  # Trigger Isolated (when doing any kind of build)
+  - name: trigger-isolated
+    taskRef:
```

Comment:
I think you'll have to add a when clause in here otherwise, the task above will run if the branch is main, and then this will run also. I think add a:
when:
  - input: "$(params.toBranch)"
        operator: in
        values: ["release"]